assert.js

```diff
@@ -130,6 +130,9 @@ assert.AssertionError = function AssertionError(options) {
   this.expected = options.expected;
   this.operator = options.operator;
   if (options.message) {
+    if (options.message instanceof Function) {
```

Comment:
`instanceof` is unreliable and shouldn't be used, what you want here is
```suggestion
    if (typeof options.message === 'function') {
```